scenes/test_string.txt

```diff
@@ -174,57 +174,57 @@
 *gosub_scene cslib_string extract_tokens str_buffer "|" "tokens"
 *comment There are actually 6 values, but only 5 are used.
 *comment The user may check for this condition.
-*gosub test_finish 6 cslib_ret
-*gosub test_finish "A" tokens_1
-*gosub test_finish "B" tokens_2
-*gosub test_finish "C" tokens_3
-*gosub test_finish "D" tokens_4
-*gosub test_finish "E" tokens_5
-*gosub test_finish 5 tokens_max
+*gosub test_assert_equal 6 cslib_ret
+*gosub test_assert_equal "A" tokens_1
+*gosub test_assert_equal "B" tokens_2
+*gosub test_assert_equal "C" tokens_3
+*gosub test_assert_equal "D" tokens_4
+*gosub test_assert_equal "E" tokens_5
+*gosub test_assert_equal 5 tokens_max
```

Comment:
Again, I assume you meant _count?

Reply:
I'm checking both, count is the next line. Better safe than sorry.